std_msgs/Header header
geometry_msgs/Pose[] poses
float32 current